### DISCUSSION TITLE
Guest user - missing trip data

### DISCUSSION THREAD

**Discussion by eili77** [tillsteinbach/VWsFriend#714] [General]
Hi,
I noticed, that trip information is not being logged/recorded when car is driven by a guest (not primary) user.
The Volkswagen App is also showing "limited service" due to privacy protection.
Is there a way to log data with VWsFriend, whle the car is driven by a **guest** user?

**Comment by tillsteinbach** [comment]
Is this still a problem? VW announced some changes beginning of the week.
